src/components/table/table.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseComponent, BaseElement, BaseOptions } from '../base';
+import { FormioComponents } from '../components';
+import { FormioTemplate } from '../../formio.template';
+import { FormArray, FormGroup, FormControl } from '@angular/forms';
+
+export interface TableOptions extends BaseOptions<any> {
+    numRows: number,
+    numCols: number,
+    rows?: Array<any>,
+    components?: Array<any>
+}
+
+export class TableComponent extends BaseComponent<TableOptions> {
+    getControl(): FormArray | FormGroup | FormControl {
+        if (!this.control) {
+            this.control = new FormArray([
+                new FormGroup({})
+            ]);
+        }
+        return this.control;
+    }
+}
+
+export class TableElement extends BaseElement<TableComponent> {
+    
```

Comment:
There is a new method in here for determining when this component is done rendering. All you need to do here is return how many components are within this component. See datagrid as an example https://github.com/formio/ng2-formio/blob/master/src/components/datagrid/datagrid.ts#L28. Looking at your template, I am thinking you will need to provide the following.

```
get numComponents() : number {
    let total = 0;
    for (let i in this.component.settings.rows) {
        total += this.component.settings.rows[i].components.length;
    }
    return total;
}
```

Please test this though, and verify that the ```onRender``` method is called in the application.